src/app/screens/ConfirmSignMessage/index.tsx

```diff
@@ -101,12 +106,13 @@ function ConfirmSignMessage() {
               </div>
             */}
           </div>
-          <ConfirmOrCancel
-            disabled={loading}
-            loading={loading}
-            onConfirm={confirm}
-            onCancel={reject}
-          />
+          <form onSubmit={handleSubmit}>
```

Comment:
nit: In this case we do not really send anything, right? It's not really a form but to align everything it might be ok, wdyt?

Reply:
Yeah, I think it will be nice to keep it consistent.